textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/common/JoinPredicate.java

```diff
@@ -0,0 +1,52 @@
+package edu.uci.ics.textdb.dataflow.common;
+
+import edu.uci.ics.textdb.api.common.Attribute;
+import edu.uci.ics.textdb.api.common.IPredicate;
+
+public class JoinPredicate implements IPredicate {
+	
+	//Reserved for future
+//	private enum CompOp {
+//		NO_OP,	//No comparision
+//		EQ_OP,	//==
+//		NEQ_OP,	//!=
+//		LT_OP,	//<
+//		GT_OP,	//>
+//		LTEQ_OP,//<=
+//		GTEQ_OP	//>=
+//	}
+	
+	private Integer outerAttributeId;
```

Comment:
Add good comments with illustrative examples to explain this Join operator and the JoinPredicate.